coalib/bearlib/__init__.py

```diff
@@ -60,7 +60,7 @@ def deprecate_settings(**depr_args):
      ['old']
 
```

Comment:
the commit shortlog could be clearer, as in:
``bearlib/__init__.py: Fix typo in docstring``